README.md

```diff
@@ -60,7 +60,7 @@ sudo -E ./beyla
 
 Now, you should see metrics on [http://localhost:9400/metrics](http://localhost:9400/metrics).
 
-See [Documentation](https://grafana.com/docs/beyla/) and the [tutorials](docs/sources/tutorial/_index.md) for more info.
+See [Documentation](https://grafana.com/docs/beyla/) and the [tutorials](https://grafana.com/docs/beyla/latest/tutorial/) for more info.
```

Comment:
Just a minor comment on the tutorial link, should we point to the 'next' docs version instead, since that reflects the state of the GitHub main branch: https://grafana.com/docs/beyla/next/tutorial?

Same for the main link, I think it's better to be https://grafana.com/docs/beyla/next/

Reply:
I'd say that it depends on what is the expected audience of this README: a developer or someone willing to use the last dev version of Beyla? or just someone that googled "grafana beyla" and landed here?

Reply:
Good point! So maybe we should be explicit: "See the latest stable release [Documentation...."?